apps/payments/nextjs/package.json

```diff
@@ -36,6 +36,8 @@
     "devDependencies": {
         "@types/react": "18.2.21",
         "@types/react-dom": "18.2.7",
+        "@zerodev/passkey-validator": "5.4.0",
```

Comment:
Could you pls make a ticket to investigate and fix later and include it in this file (ie. WAL-XXX)

Reply:
Ticket made, WAL-2732!